doc/news/astropy.rst

```diff
@@ -0,0 +1,4 @@
+**Changed:**
+
+* Changed maximum astropy version to 7.x.
+* Changed ubuntu version for build doc to 24.04.
```

Comment:
```suggestion
* Changed Ubuntu version for building the documentation in GitHub workflows to 24.04.
```

Reply:
Do we indicate changes in the repository that are not relevant to the functionality of the module?

Reply:
Probably not.